docs: Improve cron tasks and fix link in documentation (#2708)
docker-mailserver/docker-mailserver: docs/content/faq.md

    # This assumes you're having `environment: ONE_DIR=1` in the env-mailserver,
    # with a consolidated config in `/var/mail-state`
    #

>     # '> /dev/null' to send error notifications from 'stderr' to 'postmaster@example.com'

Reviewer: It's technically meant to omit `stdout` to avoid reports that are unnecessary, thus only `stderr` would trigger reports.

    # This assumes you're having `environment: ONE_DIR=1` in the env-mailserver,
    # with a consolidated config in `/var/mail-state`
    #

>     # Adding `MAILTO=""` prevents sending messages
>     # to email after completing cron task

Reviewer: ```suggestion
    # Adding `MAILTO=""` prevents cron emailing notifications of the task outcome each run
```

    #
    # Everyday 2:00AM, learn spam from a specific user
    # spam: junk directory
    0  2 * * * root  sa-learn --spam /var/mail/example.com/username/.Junk --dbpath /var/mail-state/lib-amavis/.spamassassin > /dev/null
    # ham: archive directories
    15 2 * * * root  sa-learn --ham /var/mail/example.com/username/.Archive* --dbpath /var/mail-state/lib-amavis/.spamassassin > /dev/null
    # ham: inbox subdirectories
    30 2 * * * root  sa-learn --ham /var/mail/example.com/username/cur* --dbpath /var/mail-state/lib-amavis/.spamassassin > /dev/null
    #
    # Everyday 3:00AM, learn spam from all users of a domain
    # spam: junk directory
    0  3 * * * root  sa-learn --spam /var/mail/not-example.com/*/.Junk --dbpath /var/mail-state/lib-amavis/.spamassassin > /dev/null
    # ham: archive directories
    15 3 * * * root  sa-learn --ham /var/mail/not-example.com/*/.Archive* --dbpath /var/mail-state/lib-amavis/.spamassassin > /dev/null
    # ham: inbox subdirectories

>     30 3 * * * root  sa-learn --ham /var/mail/not-example.com/*/cur* --dbpath /var/mail-state/lib-amavis/.spamassassin > /dev/null

Reviewer: Should a comment be added at the top of the config like `MAILTO=""` but noting that `> /dev/null` is to only notify the admin by mail when there is `stderr` output?

Author: Initially, I didn't see the point in this, but why not
